src/labkey/filter/Types.ts

```diff
@@ -231,7 +233,7 @@ export function getFilterTypesForType(jsonType: JsonType, mvEnabled?: boolean):
  */
 export function registerFilterType(
     displayText: string, displaySymbol?: string, urlSuffix?: string,
-    dataValueRequired?: boolean, multiValueSeparator?: string, longDisplayText?: string,
+    dataValueRequired?: boolean, labkeySqlOperator?: string, multiValueSeparator?: string, longDisplayText?: string,
```

Comment:
This will break existing usages of this method for any clients that are utilizing this method. We need to make this a backwards compatible change. This is exported in `@labkey/api` as well as accessible on LKS pages as `LABKEY.Filter.registerFilterType()`.

Reply:
good point about client usages. updated